oak_restricted_kernel/src/mm/mod.rs

```diff
@@ -38,6 +38,13 @@ mod encrypted_mapper;
 pub mod frame_allocator;
 pub mod page_tables;
 
+/// The start of kernel memory.
+pub const KERNEL_OFFSET: u64 = 0xFFFF_FFFF_8000_0000;
```

Comment:
Move the `DIRECT_MAPPING_OFFSET` here as well so that the constants would be together.